Sources/Benchmark/BenchmarkSetting.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum BenchmarkSetting {
+    case iterations(Int)
```

Comment:
Can we think through the exact meaning of this setting (and subsequently document it)? Concretely, I suspect we by default will want to auto-detect the number of iterations based on how long a warm-up iteration takes; is that a different setting (e.g. `.autoDetectIterations`) that will become the new default?

Reply:
Once we implement auto-detection of the number iterations we drop the default setting for `iterations`. Then the logic is going to be as following:

1. If the `iterations` setting was provided at benchmark or suite level: run exactly that number of iterations.
2. Otherwise dynamically detect desirable number of iterations based on a heuristic like in google/benchmark. 

I will document this once we implement auto-detection in the follow-up PR.

Reply:
SGTM